src/types.ts

```diff
@@ -30,14 +30,32 @@ export interface Options {
   usage?: boolean;
 }
 
+interface Scenario {
```

Comment:
I will suggest to use the types defined by cucumber for this: 
https://github.com/cucumber/cucumber-js/blob/main/src/support_code_library_builder/types.ts#L11